statvar_imports/india_rbistatedomesticproduct/manifest.json

```diff
@@ -0,0 +1,26 @@
+{
+    "import_specifications": [
+      {
+        "import_name": "India_RBIStateDomesticProduct",
+        "curator_emails": [
+          "support@datacommons.org"
+        ],
+        "provenance_url": "https://www.rbi.org.in/Scripts/AnnualPublications.aspx?head=Handbook%20of%20Statistics%20on%20Indian%20States",
+        "provenance_description": "RBI State Domestic Product data",
+        "scripts": [
+          "rbi_sdp_download.py",
+          "../../tools/statvar_importer/stat_var_processor.py --input_data=source_files/*.xlsx --pv_map=state_domestic_product_pv_map.csv --config_file=state_domestic_product_metadata.csv --places_resolved_csv=state_domestic_product_places_resolved.csv --output_path=test_data/state_domestic_product_output"
```

Comment:
pls change output path to a new folder `output`
`--output_path=output/state_domestic_product_output`